### ISSUE TITLE
failed install igraph

### ISSUE THREAD

**Issue by andybeet** [igraph/rigraph#338]
R version 3.5.2 (2018-12-20)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: CentOS release 6.10 (Final)

Matrix products: default
BLAS: /usr/lib64/R/lib/libRblas.so
LAPACK: /usr/lib64/R/lib/libRlapack.so

Error message:

In file included from bliss.cc:20:
bliss/graph.hh:326: error: ISO C++ forbids initialization of member ‘refine_equal_to_first’
bliss/graph.hh:326: error: making ‘refine_equal_to_first’ static
bliss/graph.hh:326: error: ISO C++ forbids in-class initialization of non-const static member ‘refine_equal_to_first’
make: *** [bliss.o] Error 1
ERROR: compilation failed for package ‘igraph’
* removing ‘/net/home5/abeet/R/x86_64-redhat-linux-gnu-library/3.4/igraph’
Error in i.p(...) : 
  (converted from warning) installation of package ‘igraph’ had non-zero exit status

Not sure what this means.

**Comment by dongjoban17** [comment]
Also running into the same issue with the exactly the same error message. There was a thread [here](https://community.rstudio.com/t/non-zero-exit-status-while-updating-igraph-package-version-on-rstudio-server-on-linux-cent-os-6-server/29114/2) as well, but nothing too helpful as it was never resolved.

**Comment by andybeet** [comment]
@dongjoban17 that worked. Thanks! Temporary fix until i can chat to IT regarding updating compilers.

**Comment by ppdouble** [comment]
when I compare igraph 1.2.3 and igraph 1.2.4 from [cran igraph archive](https://cran.r-project.org/src/contrib/Archive/igraph/)
```
$diff graph.hh ../../../igraph124/src/bliss/graph.hh
326c326
<   bool refine_equal_to_first;
---
>   bool refine_equal_to_first = false;
```

```
$ diff graph.cc ../../../igraph124/src/bliss/graph.cc
611c611
<   bool needs_long_prune;
---
>   bool needs_long_prune = false;
```

Apparently, it should not initialize the variable `refine_equal_to_first` in this way.

Install igraph `1.2.4`, with environment `R version 3.2.0 (2015-04-16) -- "Full of Ingredients"` `Red Hat Enterprise Linux Server release 6.3 (Santiago) x86_64`

```
In file included from bliss.cc:20:
bliss/graph.hh:326: error: ISO C++ forbids initialization of member ‘refine_equal_to_first’
bliss/graph.hh:326: error: making ‘refine_equal_to_first’ static
bliss/graph.hh:326: error: ISO C++ forbids in-class initialization of non-const static member ‘refine_equal_to_first’
make: *** [bliss.o] Error 1
ERROR: compilation failed for package ‘igraph’
```

I am using `R CMD INSTALL -l $R_LIBS ../../igraph_1.2.3.tar.gz` to install igraph. But function `igraph_version()` give me the edition number as `1.2.2`.

**Comment by ntamas** [comment]
This is now fixed in igraph 1.3.0, which updated the C core of igraph to 0.9.7.
